book/6-computer-vision.tex

```diff
@@ -5,4 +5,71 @@ \chapter{Computer Vision}
 \clearpage
 \thispagestyle{cvstyle}
 \section{Computer Vision Sample Metric}
-\subsection{Computer Vision Sample Metric}
\ No newline at end of file
+\subsection{Computer Vision Sample Metric}
+
+
+% ---------- PSNR ----------
+\clearpage
+\thispagestyle{cvstyle}
+\section{PSNR}
+\subsection{Peak Signal-to-Noise Ratio}
+
+In image generation and reconstruction, it is important to measure how closely generated images resemble reference images. 
+PSNR is a commonly used metric for comparing pixel differences between them.
+
+% equation
+\begin{center}
```

Comment:
Flip the arrows of MAX and R(i, j) so they look nicer.

The word width should be at the end of the arrow

<img width="597" alt="Screenshot 2024-10-17 at 11 19 21" src="https://github.com/user-attachments/assets/879c667b-948d-4daa-bc26-3b5facc2ba22">